RULES.md

```diff
@@ -136,39 +136,62 @@ There are no warnings for this rule pack.
+| [NIST.800.53-ALBHttpDropInvalidHeaderEnabled](https://docs.aws.amazon.com/config/latest/developerguide/alb-http-drop-invalid-header-enabled.html)       | The Application Load Balancer does not have invalid http header dropping enabled - (Control ID: AC-17(2))                                                                                 | Ensure that your Application Load Balancers (ALB) are configured to drop http headers. Because sensitive data can exist, enable encryption in transit to help protect that data.                                                                                                                                                                                                                                                                                 |
+| [NIST.800.53-ALBLoggingEnabled](https://docs.aws.amazon.com/config/latest/developerguide/elb-logging-enabled.html)                                      | The Application Load Balancer does not have logging enabled - (Control ID: AU-2(a)(d))                                                                                                    | Application Load Balancing activity is a central point of communication within an environment. Ensure ALB logging is enabled. The collected data provides detailed information about requests sent to the ELB. Each log contains information such as the time the request was received, the client\'s IP address, latencies, request paths, and server responses.                                                                                                |
+| [NIST.800.53-APIGWCacheEnabledAndEncrypted](https://docs.aws.amazon.com/config/latest/developerguide/api-gw-cache-enabled-and-encrypted.html)           | The API Gateway stage does not have caching enabled and encrypted. - (Control IDs: SC-13, SC-28)                                                                                          | To help protect data at rest, ensure encryption is enabled for your API Gateway stage\'s cache. Because sensitive data can be captured for the API method, enable encryption at rest to help protect that data.                                                                                                                                                                                                                                                  |
```

Comment:
```suggestion
| [NIST.800.53-APIGWCacheEnabledAndEncrypted](https://docs.aws.amazon.com/config/latest/developerguide/api-gw-cache-enabled-and-encrypted.html)           |  The API Gateway stage does not have caching enabled and encrypted for all methods - (Control IDs: SC-13, SC-28)                                                                                          | To help protect data at rest, ensure encryption is enabled for your API Gateway stage\'s cache. Because sensitive data can be captured for the API method, enable encryption at rest to help protect that data.                                                                                                                                                                                                                                                  |
```